MalcolmMark/Free-Mentors
# #168237714 Ft(server) up date user to mentor

#### What does this PR do?
Update a user to a mentor
#### Description of Task to be completed?
admin should be able to change a user to a mentor
#### How should this be manually tested?
first run the server 
then after a successful run check if the port is live at 3000
you can later go to postman and test it locally
#### Any background context you want to provide?
N/A
#### What are the relevant pivotal tracker stories?
#168237714
#### Screenshots (if appropriate)
![pppppppppppppppppppg](https://user-images.githubusercontent.com/45765235/64257855-6fa06400-cf26-11e9-8941-410b0ef317b1.PNG)

#### Questions:
N/A
